frontend/webapp/components/overview/sources/sources.manage.list/sources.manage.list.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ManagedListWrapper, EmptyListWrapper } from "./sources.manage.styled";
+import Empty from "@/assets/images/empty-list.svg";
+import SourceManagedCard from "./sources.manage.card";
+
+export function SourcesManagedList({ data = [1, 1, 1, 1] }) {
+  function renderDestinations() {
+    return data.map((source: any) => <SourceManagedCard />);
+  }
+
+  return (
+    <>
+      <ManagedListWrapper>
+        {data?.length === 0 ? (
```

Comment:
id `data` is `undefined` this condition will be falsy and we will fallback to render the list which will throw